server/src/test/java/keywhiz/service/permission/PermissionCheckTest.java

```diff
@@ -0,0 +1,51 @@
+package keywhiz.service.permission;
+
+import com.codahale.metrics.MetricRegistry;
+import com.google.inject.Provider;
+import keywhiz.service.permissions.Action;
+import keywhiz.service.permissions.AlwaysAllowPermissionCheck;
+import keywhiz.service.permissions.AlwaysFailPermissionCheck;
+import keywhiz.service.permissions.KeywhizPrincipal;
+import keywhiz.service.permissions.PermissionCheck;
+import keywhiz.service.permissions.PermissionCheckImpl;
+import keywhiz.service.permissions.PermissionCheckProvider;
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import org.mockito.Mock;
+import org.mockito.junit.MockitoJUnit;
+import org.mockito.junit.MockitoRule;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class PermissionCheckTest {
```

Comment:
Since PermissionCheck is an interface it doesn't have any behavior to test (default methods aside), so it probably doesn't need its own unit test. Looking at the test names, this is probably supposed to be AlwaysAllowPermissionCheckTest.